packages/govuk-frontend/babel.config.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Babel config
+ *
+ * @type {import('@babel/core').ConfigFunction}
+ */
+module.exports = function (api) {
+  const browserslistEnv = !api.env('test')
+    ? 'javascripts'
+    : 'node'
+
+  return {
+    presets: [
+      ['@babel/preset-env', {
+        browserslistEnv,
+        loose: browserslistEnv === 'javascripts',
```

Comment:
Could we include a brief rationale  for using `loose` here? Babel documentation links to an article which recommends against using it, and also suggests migrating to `assumptions`, so it'd be good to have something here helping future devs understand why we're doing it.

Reply:
Yeah I'll add a comment

I picked `loose: true` for browsers as the `assumptions` options are fairly huge:
https://github.com/babel/rfcs/blob/main/rfcs/0003-top-level-assumptions.md#assumptions-list

We generally want the smallest KB code possible

For example, if we added [private class fields](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields) Babel would assume [`{ privateFieldsAsProperties: true }`](https://babeljs.io/docs/assumptions#privatefieldsasproperties) was good enough, rather than trying to hide away private things into a [`WeakMap`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakMap) with polyfills

>Assume that "soft privacy" is enough for private fields, and thus they can be stored as public non-enumerable properties with an unique name (rather than using an external `WeakMap`). This makes debugging compiled private fields easier.